userscript-header.js

```diff
@@ -0,0 +1,13 @@
+// ==UserScript==
+// @name         Webhallen stats & comparisons
+// @namespace    Webhallen
+// @version      $USER_SCRIPT_VERSION
+// @description  $USER_SCRIPT_DESC
+// @author       $USER_SCRIPT_AUTHORS
+// @match        https://www.webhallen.com/*
+// @icon         https://www.google.com/s2/favicons?sz=64&domain=webhallen.com
+// @updateURL    https://github.com/Schanihbg/webhallen-sum-userscript/releases/latest/download/webhallen.user.js
+// @downloadURL  https://github.com/Schanihbg/webhallen-sum-userscript/releases/latest/download/webhallen.user.js
+// @grant        GM_addStyle
```

Comment:
This PR consolidates the userscripts to one, so they share grants.

IMHO it seemed more convenient from a user perspective to bundle them together rather than installing and updating a handful of separate userscripts. I can tweak the build process to produce multiple scripts if that's preferable; WDYT?

(cc @Schanihbg)

Reply:
Having one is simpler but I think having separate ones gives users the choice to only enable the ones that are relevant to them. I don’t think everyone wants for example the category fix or compare products scripts.

Reply:
If I could choose, I like the idea of having some sort of settings page/section where you can toggle the various features. It'd also open up the possibility of having other configurables.

Reply:
Made a separate PR onto my main branch with userscript settings that append themselves on the regular profile settings page: https://github.com/Sleavely/webhallen-sum-userscript/pull/2

Reply:
I think we can merge it now right?